engine/server/sv_game.c

```diff
@@ -3156,11 +3164,22 @@ void SV_AllocStringPool( void )
 void SV_FreeStringPool( void )
 {
 #ifdef XASH_64BIT
+	size_t pagesize;
```

Comment:
You defined pagesize here, set it's value and not used it.

Is it meant to use pagesize instead of sysconf( _SC_PAGESIZE ) in 3180? Does USE_MMAP defined on Win64 at all?
engine/common/filesystem.c

Reply:
USE_MMAP is not defined on Win64.
Yeah... it may be useless and I will remove it.

Reply:
On Win64, XASH_64BIT should be defined. So munmap should be ignored.
I have seen USE_MMAP before, so I added this check